www/docs/nextjs/nextjs-intro.md

```diff
@@ -0,0 +1,166 @@
+---
+id: intro
+title: Usage with Next.js
+sidebar_label: Usage with Next.js
+slug: /nextjs
+---
+
+tRPC and Next.js are a match made in heaven! Next.js makes it easy for you to build your client and server together in one codebase. This makes it easy to share types between them.
+
+tRPC includes dedicated tools to make the Next.js developer experience as seamless as possible.
+
+### Recommended file structure
+
+Recommended but not enforced file structure. This is what you get when starting from the examples.
+
+```txt
+├── pages
+│   ├── _app.tsx # <-- wrap App with `withTRPC()`
+│   ├── api
+│   │   └── trpc
+│   │       ├── [trpc].ts # <-- tRPC response handler
+│   │       └── [...] # <-- potential sub-routers
+│   └── [...]
+├── trpc # <-- (optional) if prisma is added
```

Comment:
```
├── trpc # your sub-routers
```

Reply:
Adding these files under `pages` would mess things up since Next.js would interpret them as new API routes. So if you added a `user.ts` file containing a user router, all calls to `/api/trpc/user` would result in an internal server error. Anything that isn't a page should live elsewhere.